server/app.js

```diff
@@ -78,7 +78,7 @@ app.use((req, res) => {
 
 // error handler
 /* istanbul ignore next  */
-app.use((err, req, res, next) => {
+app.use((err, req, res) => {
```

Comment:
This actually changes the behavior from being an error handler to a request handler, so now `err` is being passed the Request and not the error as before.

While the `next` callback function was unused, [Express detects four parameters and treats the function differently](https://expressjs.com/en/guide/error-handling.html#writing-error-handlers), so we need it.

A common pattern in JavaScript is to use `_` for unused variables:
```javascript
app.use((err, req, res, _) => {
```

and to customize ESLint to follow this pattern, we can add this to the `rules` object in `.eslintrc.js`:

```javascript
{
  rules: {
    "no-unused-vars": ["warn", { argsIgnorePattern: "^_$" }]
  }
}
```

***

Otherwise, it wouldn't be completely unreasonable to ignore this particular unused variable explicitly:
```javascript
/* eslint-disable no-unused-vars */
app.use((err, req, res, next) => {
```